source/develop/release-management/features/engine/metrics-store.html.md

```diff
@@ -53,6 +53,105 @@ Once you have finished this step, you should have:
 
   * Kibana - <https://kibana.{hostname}>
   * OpenShift portal - <https://openshift.{hostname}>
+  
+  
+
+**Update mux pod resources**
+
+1. ssh into the logging machine and use the command line.  
+**Note:** You can probably do all of this with the OpenShift UI too - https://logging-machine:8443
+
+2. Run
+
+       # $ oc edit dc logging-mux
+
+Edit the section
+
+      spec:
+        template:
+          spec:
+            containers:
+      ...
```

Comment:
this is an ellipsis `...` which in this context means 'there may be a lot more that follows this, but skip it until you get to this next part'